src/packages/DAO/elections.dao.ts

```diff
@@ -48,7 +48,7 @@ export async function startElection(formData: FormData) {
     },
   });
 
-  revalidatePath("/");
+  redirect(`http://localhost:3000/elections/${electionId}`);
```

Comment:
We need to setup this as a environment var called APP_BASE_URL as this will change depending on environment.

With the redirect command, in this case here, you can also omit the base url and send only the paths ahead, like:
```suggestion
  redirect(`elections/${electionId}`);
```

Reply:
No need for adding the env var as I mentioned if you follow this alternative

Reply:
change here [2b69f27](https://github.com/podcodar/voting-system/pull/65/commits/2b69f271419a5a16984b86c5ad2d7b916ba615a7)